security-groups/spg_deprecated.tf

```diff
@@ -98,6 +98,18 @@ resource "aws_security_group_rule" "ssh_jenkins_in" {
   description       = "TF - ssh_jenkins_in"
 }
 
+resource "aws_security_group" "amazonmq_in" {
```

Comment:
could we move this to its own file called amazon-mq.tf

these other SGs are deprecated as now live in spg project